EssentialsDiscord/src/main/resources/config.yml

```diff
@@ -0,0 +1,250 @@
+    # You must restart your server after changing this
+    enabled: true
+    # Whether or not the command should be hidden from other people in the channel
+    # If set to false, members of the discord will be able to see the target and content of your message
+    hide-command: true
+    # List of user ids or role names/ids allowed to use this command (or '*' to allow anyone to access it)
+    allowed-roles:
+      - "*"
+    # List of user ids or role names/ids who can message vanished players or players who disable messages. If '*' is
+    # used, all people on discord would be allowed to message vanished players (and therefore expose they are actually online)
+    # and message players who disable messages
+    admin-roles:
+      - "Admins"
+      - "123456789012345678"
+  list:
+    # Set to false if you do not want this command to show up on the discord command selector
+    # You must restart your server after changing this
+    enabled: true
+    # Whether or not the command should be hidden from other people in the channel
+    # If set to false, members of the discord will be able to see when you use this command
+    hide-command: true
+    # List of user ids or role names/ids allowed to use this command (or '*' to allow anyone to access it)
+    allowed-roles:
+      - "*"
+    # List of user ids or role names/ids who can see vanished players in the player list. If '*' is used, all people
+    # on discord would be allowed to see vanished players (and therefore expose they are actually online)
+    admin-roles:
+      - "Admins"
+      - "123456789012345678"
+
+# Whether or not links to attachments in discord messages should be displayed in chat or not
+# If this is set to false and a message from discord only contains an image/file and not any text, nothing will be sent
+show-discord-attachments: true
+
+# A list of roles allowed to send minecraft color/formatting codes from discord to MC
+# This applies to all aspects such as that discord->mc chat relay as well as commands
+# You can either use '*' (for everyone), a role/id, or just a user id
+permit-formatting-roles:
+  - "Admins"
+  - "Color Codes"
+
+# The presence of the bot, including its status, activity and status message
+presence:
+  # The online status of the bot. Must be one of the following;
+  # * "online" - Shows as green circle (Online)
+  # * "idle" - Shows as yellow half-circle (Away)
+  # * "dnd" - Shows as red circle (Do Not Disturb)
+  # * "invisible" - Makes the bot appear offline
+  status: online
+  # The activity of the bot to be prefixed before your message below. Must be one of the following;
+  # * "playing" - Shows up as "Playing <message>"
+  # * "listening" - Shows up as "Listening to <message>"
+  # * "watching" - Shows up as "Watching <message>"
+  # * "competing" - Shows up as "Competing in <message>"
+  # * "custom" - Just shows the message defined below with no prefix
```

Comment:
Custom Status is read-only for bots. They can't use them.
If they were usable would JDA already have a dedicated `custom` method or similar. I would recommend to not support this until there is official documentation about it being usable by bots either on Discord's Docs, or in the JDA Javadoc.

Reply:
Thanks for pointing that out